docs/install/prerequisites.md

```diff
@@ -1,19 +1,19 @@
 # Prerequisites
 
 - Linux kernel version >= 4.18
-- Linux kernel headers available under conventional location (see [Linux Headers](#TODO) section for more info)
-- libc, and the libraries: libelf, zlib
```

Comment:
libbpf is not a prerequisite

Reply:
@itaysk are you saying that because we package libbpf source code with this repo?

Would all these library prerequisites be eliminated if we statically compile our releases? And headers once I finish with CO:RE?

Reply:
if someone wants to run tracee, they can download the binary, and run it, they don't need to have libbpf in order for tracee to run, right? the same can't be said for libelf, zlib which are required at runtime.

Yes I think if we statically compile tracee those reqs would be eliminated